hca/util/__init__.py

```diff
@@ -420,7 +420,8 @@ def _get_jwt_from_service_account_credentials(self):
                    'exp': exp,
                    'email': service_credentials["client_email"],
                    'scope': ['email', 'openid', 'offline_access'],
-                   'https://auth.data.humancellatlas.org/group': 'hca'
+                   'https://auth.data.humancellatlas.org/group': 'hca',
+                   'https://auth.data.humancellatlas.org/email': service_credentials["client_email"]
```

Comment:
Do we intend for the email field to eventually become `https://auth.data.humancellatlas.org/email` instead, similar to the way the group field is `https://auth.data.humancellatlas.org/group`?

Reply:
No, the email field only exists when google service accounts are used and doesn't appear when 
JWT are minted from auth0. This is why we need the additional `https://auth.data.humancellatlas.org/email` field.